packages/schema/src/config/common.ts

```diff
@@ -342,15 +351,19 @@ export default defineUntypedSchema({
    * inside the `ignore` array will be ignored in building.
    */
   ignore: {
-    $resolve: async (val, get) => [
-      '**/*.stories.{js,cts,mts,ts,jsx,tsx}', // ignore storybook files
-      '**/*.{spec,test}.{js,cts,mts,ts,jsx,tsx}', // ignore tests
-      '**/*.d.{cts,mts,ts}', // ignore type declarations
-      '**/.{pnpm-store,vercel,netlify,output,git,cache,data}',
-      relative(await get('rootDir'), await get('analyzeDir')),
-      relative(await get('rootDir'), await get('buildDir')),
-      await get('ignorePrefix') && `**/${await get('ignorePrefix')}*.*`
-    ].concat(val).filter(Boolean)
+    $resolve: async (val, get) => {
+      const rootDir = await get('rootDir')
+      const ignorePrefix = await get('ignorePrefix')
```

Comment:
Small perf we could use parallel promises: `const[rootDir, ignorePrefix] = await Promise.all(get('rootDir'), get('ignorePrefix'))`

Reply:
Oh you're right! Thanks, I missed that. I know another place this can be added in.

Reply:
Is there a way to "append" commits to a merged PR or should I open a new one?

Reply:
You probably need a new one..